sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFileLinesReader.scala

```diff
@@ -46,7 +46,7 @@ class HadoopFileLinesReader(
 
   def this(file: PartitionedFile, conf: Configuration) = this(file, None, conf)
 
-  private val iterator = {
+  private val _iterator = {
```

Comment:
This is to resolve a weird compile error:

```
[ERROR] [Error] /Users/seanowen/Documents/spark_2.13/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFileLinesReader.scala:49: weaker access privileges in overriding
final def iterator: Iterator[org.apache.hadoop.io.Text] (defined in trait Iterator)
  override should not be private
[ERROR] [Error] /Users/seanowen/Documents/spark_2.13/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFileWholeTextReader.scala:38: weaker access privileges in overriding
final def iterator: Iterator[org.apache.hadoop.io.Text] (defined in trait Iterator)
  override should not be private
```
Not sure, but easy to avoid the name conflict entirely.